sdk/device_test.go

```diff
@@ -183,7 +183,7 @@ func TestDevice_JSON_1(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(
 		t,
-		`{"Kind":"","Metadata":null,"Plugin":"","Info":"","Location":null,"Data":null,"Outputs":null,"SortOrdinal":0}`,
```

Comment:
The string order delta here is part of the golang map ordering sorta-kinda-random business. The fields are the same as previously.

Thanks a lot Larry.